requirements.txt

```diff
@@ -1,4 +1,4 @@
-Django>=3.2
+Django>=3.2,<4.0
```

Comment:
You need to remove this restriction: <4.0
It is causing the CI/CD pipeline to fail to install Django.
We will fix the rundemo feature in the near future but the project must support Django 4 now.

Reply:
Jos is done